src/features/defi/providers/univ2/hooks/useUniV2LiquidityPool.ts

```diff
@@ -284,7 +258,7 @@ export const useUniV2LiquidityPool = ({
       return uniswapRouterContract.interface.encodeFunctionData('removeLiquidity', [
         asset0ContractAddress,
         asset1ContractAddress,
-        lpAmountBaseUnit,
+        toBaseUnit(lpAmount, lpAsset.precision),
```

Comment:
Can we bring the base unit / precision vernacular back for `lpAmount` ?

Reply:
`lpAmount` wasn't changed, just doing an inline `toBaseUnit` call instead of a separate variable named `lpAmountBaseUnit = toBaseUnit(lpAmount, lpAsset.precision)`

Reply:
Yeah I guess it would make sense to LSP rename it to `lpAmountCryptoPrecision` now that this is inline

Reply:
Resolving this one, we can tackle as a follow-up if this is confusing